### PR TITLE
This PR adds recipes for epics-probe.

### PR DESCRIPTION
**Pull request by pthomasj**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
I'm happy to be named as a maintainer of this recipe.

**Comment by pthomasj** [comment]
@conda-forge-admin, please ping team

File: recipes/epics-probe/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export MOTIF_INC="${PREFIX}/include"
+export MOTIF_LIB="${PREFIX}/lib"
+export X11_INC="${PREFIX}/include"
+export X11_LIB="${PREFIX}/lib"
+
+_make="make -j1 V=1 VERBOSE=1 EPICS_BASE=${EPICS_BASE} TOP=${TOP}"
+
+# build
+${_make} build
+
```
Comment (ocefpaf) [inline recipes/epics-probe/build.sh:15]: Is there a make check we can run here?

Comment (pthomasj) [reply recipes/epics-probe/build.sh:15]: @ocefpaf @duncanmmacleod I don't believe so. I think the problem here may be the same as the one for epics-striptool and other epics extensions: https://github.com/conda-forge/staged-recipes/pull/18283#discussion_r825749741

File: recipes/epics-probe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - xorg-libxp  # [unix]
+    - xorg-libxt  # [unix]
+
+test:
+  commands:
+    - test -x ${PREFIX}/epics/bin/${EPICS_HOST_ARCH}/probe
+
+about:
+  home: https://epics.anl.gov/extensions/probe/index.php
+  license: EPICS
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/epics-probe/meta.yaml:44]: Argh! Bamboozled by the linter again :-(

[the remark this answers is not in the document]
Comment (jakirkham) [reply recipes/epics-probe/meta.yaml:44]: No worries

Did some follow up here ( https://github.com/conda-forge/staged-recipes/pull/18283#discussion_r826258507 )

Will file an issue on the feedstock once it is produced to track

[the remark this answers is not in the document]
Comment (pthomasj) [reply recipes/epics-probe/meta.yaml:44]: I merged a pull request to the feedstock to add the license file.

[the remark this answers is not in the document]
Comment (jakirkham) [reply recipes/epics-probe/meta.yaml:44]: Thanks!

For reference this is the PR ( https://github.com/conda-forge/epics-probe-feedstock/pull/1 )

**Pull request merged**